api/proto/info/info.proto

```diff
@@ -1084,4 +1084,8 @@ message ZInfoLocation {
   // Vertical position uncertainty in meters.
   // Negative values are not valid and represent unavailable uncertainty.
   float vertical_uncertainty = 8;
+  // Logical label of the device used to obtain this location information.
+  // If the GNSS receiver is integrated with an LTE modem, then this is the logical
+  // label from the modem's PhysicalIO entry.
+  string logicallabel = 9;
```

Comment:
This looks good to me. But the uncertainty, reliability fields do not make much sense to zedcloud. I'm not even sure if zedcloud should even store them. Can EVE take care of filtering the readings sent to zedcloud?

Reply:
@gkodali-zededa if a customer cares about a device location presumably they also care whether the uncertainty is 10 meters or 10 kilometers, and based on my experience the uncertainty can vary a lot based on how many satelites the GNSS receiver gets a fix on.

Reply:
If it's 10kms, I don't see a meaning of even presenting such coordinates. Don't send such coordinates. If it's too hard for individual EVE devices (with more information about the sensor itself) to ascertain such information, it is only going to make it worse for zedcloud.

Reply:
I have added an improvement to how we publish/filter location updates that will favor updates with higher accuracy (measurements can jump quite a bit): https://github.com/milan-zededa/eve/blob/qmicli/pkg/pillar/dpcmanager/wwan.go#L269-L289
But otherwise I do not see a strong need to filter published data any further. By default we will publish only 24 times a day from each device. Stripping 4 float/enum fields will make only negligible difference. Also skipping updates if they are deem too inaccurate could lead to very infrequent publishing.
Finally, I wouldn't want to make too many assumptions here about the customer usage - zedcloud might not even be the "last" controller. It could be integrated into a bigger system where uncertainty/reliability information might be useful to process/display.